### PR TITLE
Switch to `nextest --retries` to spot flaky tests

### PR DESCRIPTION
**Pull request by yvan-sraka**
https://nexte.st/book/retries.html

### CODE REVIEW

**Review by AureliaDolo — COMMENTED** [review]
Is the number of attempts limited?

**Comment by yvan-sraka** [comment]
https://twitter.com/sunshowers6/status/1493283870405316608?s=20

We could pass a number of max attempts to it, yes!

**Review by adrien-zinger — APPROVED** [review]
let's try that :)

**Comment by AureliaDolo** [comment]
> https://twitter.com/sunshowers6/status/1493283870405316608?s=20
> 
> We could pass a number of max attempts to it, yes!

You mean that for now if a test fails it will be retried indefinitely ?

**Comment by yvan-sraka** [comment]
No we have set-up it, for e.g. here https://nexte.st/book/configuration.html

**Comment by damip** [comment]
cool !

**Review by adrien-zinger — DISMISSED** [review]

**Review by AureliaDolo — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -131,7 +131,12 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - uses: actions-rs/cargo@v1
         with:
-          command: test
+          command: install
+          args: nextest
```
Comment (qdrn) [inline .github/workflows/ci.yml:135]:
```suggestion
          args: cargo-nextest
```

Comment (yvan-sraka) [reply .github/workflows/ci.yml:135]: `cargo` works this way: a binary crate named `cargo-foobar` installed in `~/.cargo` using `cargo install cargo-foobar` could be then called using `cargo foobar`

**Comment by yvan-sraka** [comment]
bors merge

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -131,7 +131,12 @@ jobs:
...
+      - uses: actions-rs/cargo@v1
+        with:
+          command: nextest
+          args: --retries 2
```
Comment (qdrn) [inline .github/workflows/ci.yml:139]:
```suggestion
          args: run --retries 2
```

**Review by qdrn — CHANGES_REQUESTED** [review]

**Comment by qdrn** [comment]
bors merge

**Review by qdrn — APPROVED** [review]

**Review by adrien-zinger — DISMISSED** [review]

**Comment by adrien-zinger** [comment]
Bors merge

**Review by qdrn — APPROVED** [review]

**Comment by qdrn** [comment]
bors merge

**Pull request merged**